server.go

```diff
@@ -2492,7 +2492,11 @@ func (s *server) Start() error {
 		// brontide.NewListener() is called in newServer. This means
 		// that we are actually listening and partially accepting
 		// inbound connections even before the connMgr starts.
+		//
+		// TODO(yy): move the log into the connMgr's `Start` method.
+		srvrLog.Info("connMgr starting...")
 		s.connMgr.Start()
+		srvrLog.Debug("connMgr started")
```

Comment:
nit: I'd suggest to have both on info level.

Reply:
think it's an implicit convention that we use `info` on starting and use `debug` on started to not spam the log tho

Reply:
TIL we have this convention :), sgtm!